src/components/Post/Post.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { AiOutlineHeart, AiOutlineComment, FiBookmark } from "assets/icons";
+import { IconButton } from "components";
+
+const Post = () => {
+  return (
+    <article className="post gutter-bottom-16">
+      <header className="post-header gutter-bottom-16">
+        <div className="post-user-details">
+          <span className="avatar avatar-round">
+            <img
+              className="responsive-img"
+              src="https://adaptiveui.netlify.app/src/assets/avatar-default.png"
+              alt="...."
```

Comment:
Please give a meaningful alt.

Reply:
for now, its just dummy data.. will add it while adding db.